x-pack/test_serverless/functional/services/svl_search_navigation.ts

```diff
@@ -22,5 +22,13 @@ export function SvlSearchNavigationServiceProvider({
         await testSubjects.existOrFail('svlSearchOverviewPage', { timeout: 2000 });
       });
     },
+    async navigateToElasticsearchStartPage() {
+      await retry.tryForTime(60 * 1000, async () => {
+        await PageObjects.common.navigateToApp('elasticsearch/start', {
+          shouldLoginIfPrompted: false,
+        });
+        await testSubjects.existOrFail('elasticsearchStartPage', { timeout: 2000 });
```

Comment:
Is 2000 long enough for on serverless Rodney?
This will be run against mki (real serverless), instrumented via qaf in the serverless ftr runs, every three hours.

Reply:
My team can help you get that setup!

Reply:
@wayneseymour can you follow-up on this? the `navigateToLandingPage()` function above this also uses a 2000 timeout and it is being used in the currently serverless mki tests without any issues that I am aware of.

Reply:
here are the timeouts for svl: 
https://github.com/elastic/kibana/blob/main/x-pack/test_serverless/shared/config.base.ts#L185